org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/AbstractADWindowContent.java

```diff
@@ -3731,13 +3731,8 @@ public void onEvent(Event event) throws Exception {
 				}
 				Executions.schedule(getComponent().getDesktop(), e -> dialog.focus(), new Event("onPostShowProcessModalDialog"));
 			}
-			if (adTabbox.getSelectedGridTab().isQuickForm()) {
-				adTabbox.getSelectedGridTab().dataRefreshAll(false, false);
-			}
-			else
-			{
-				onRefresh(true, false);
-			}
+			// no need to refresh here, it will be refreshed when the process finishes
+			// see onEvent WINDOW_CLOSE for ProcessModalDialog
```

Comment:
adding the comment here is a strange one. you would have to goes back to this exact changeset to understand what it means.

Reply:
ready - comment removed